### ISSUE TITLE
Question/Feature Request: Support for department-based Bank Holidays

### ISSUE THREAD

**Issue by filo891** [timeoff-management/application#406]
We are facing a problem, where employees from different departments within a company are spread across multiple states and have different Bank Holidays applicable to them.

Would you consider to allow overriding company Bank Holidays on the Department level (similarly as employees can now use an overridden company-wide schedule)?

**Comment by vpp** [comment]
yeah, it is frequently asked feature that has to be added: multi-location
